Android/app/src/main/java/com/example/rocketcorner/User.java

```diff
@@ -0,0 +1,72 @@
+package com.example.rocketcorner;
+
+import java.util.HashMap;
+
+public class User {
+//    private String id;
+    private String username;
+    private String email;
+    private String password;
+    private double balance;
+    private HashMap<String, Integer> cart;
+
+//    public String getId() {
+//        return id;
+//    }
+
+    public String getUsername() {
+        return username;
+    }
+
+    public void setUsername(String username) {
+        this.username = username;
+    }
+
+    public String getEmail() {
+        return email;
+    }
+
+    public void setEmail(String email) {
+        this.email = email;
+    }
+
+    public String getPassword() {
+        return password;
+    }
+
+    public void setPassword(String password) {
+        this.password = password;
+    }
+
+    public double getBalance() {
+        return balance;
+    }
+
+    public void setBalance(double balance) {
+        this.balance = balance;
+    }
+
+    public HashMap<String, Integer> getCart() {
+        return cart;
+    }
+
+    public void setCart(HashMap<String, Integer> cart) {
+        this.cart = cart;
+    }
+
+    public User getAllUsers(){
+        return this;
+    }
+
+    @Override
+    public String toString() {
+        return "User{" +
+                //"id=" + id +
+                ", username='" + username + '\'' +
+                ", email='" + email + '\'' +
+                ", password='" + password + '\'' +
+                ", balance=" + balance +
+                ", cart=" + cart +
```

Comment:
I'm not sure if calling the cart like this will work, might have to loop through and get all the values.

Reply:
I'll need to have dummy data that isn't null to test this out. If you could update the user to have a cart that isn't null that would be very helpful!

Reply:
Just saw this, user UBZR408EiKgn79QmV0Jh should have an item in their cart